interfaces/builtin/all.go

```diff
@@ -148,8 +148,14 @@ func MockInterface(iface interfaces.Interface) func() {
 
 type byIfaceName []interfaces.Interface
 
-func (c byIfaceName) Len() int      { return len(c) }
-func (c byIfaceName) Swap(i, j int) { c[i], c[j] = c[j], c[i] }
```

Comment:
This way to format is used a lot in go for the super short "swap/len" helpers that are needed for sorting custom types. It's like this in many places in the snapd code already (`git grep 'Swap(.*int) {'` will show around 30ish) so I'm inclined to just leave it because it's consistent with the go conventions and the rest of the code (even though I agree that it's a bit unusual in the context of this file).

Reply:
Noted.